src/capi_utils/monit_utils.sh

```diff
@@ -37,12 +37,20 @@ function wait_for_server_to_become_unavailable() {
 #
 function wait_for_server_to_become_healthy() {
```

Comment:
this function is also used in our post-start script, does your change alter that behavior in a significant way?

Reply:
All the semantics of the script have been preserved. The only change in behavior is the condition is satisfied after 10 successful `/v2/info` calls.

Reply:
The default threshold should be one, and y'all should specify 10 for the backup and restore scripts.